app/src/androidTest/java/com/google/samples/apps/topeka/activity/SignInActivityTest.java

```diff
@@ -65,20 +72,142 @@ public void checkFab_initiallyNotDisplayed() {
     }
 
     @Test
+    public void signIn_checkIfFirstNameEditTextIsInitiallyEmpty() {
+        onView(withId(R.id.first_name)).check(matches(withText(isEmptyOrNullString())));
+    }
+
+    @Test
+    public void signIn_checkIfLastInitialEditTextIsInitiallyEmpty() {
+        onView(withId(last_initial)).check(matches(withText(isEmptyOrNullString())));
+    }
+
+    @Test
     public void signIn_performSuccessful() {
         inputData();
-        onView(withId(R.id.done)).check(matches(isDisplayed()));
+        onView(withId(R.id.done)).check(matches(isDisplayed())).perform(click());
+    }
+
+    @Test
+    public void signIn_chooseAvatar_01() {
```

Comment:
Every time I want to use an <code>onData</code>, this shows me matching error. It has only enum and its values, no views, so I don't think it's a good idea to change <code>onView</code>